smart-common/src/main/java/org/smartdata/versioninfo/VersionInfoWrite.java

```diff
@@ -36,7 +42,7 @@
 
 public class VersionInfoWrite {
   File directory = new File("");
-  String pom = directory.getAbsolutePath() + "/" + "pom.xml";
+  String pom = directory.getAbsolutePath() + "/smart-common/pom.xml";
```

Comment:
The old code use project pom not smart-commont code? If the project can be read, maybe using it is better.

Reply:
in project pom, tag <version>'s parent tag is <project>,the root node. 
it is better use to ssm-common pom.